src/App.tsx

```diff
@@ -1,11 +1,14 @@
-import { ThemeProvider, Typography } from "@mui/material";
-import { theme } from "./theme/theme";
+import { Route, Routes } from "react-router-dom";
+import Home from "./components/Home/Home";
```

Comment:
To avoid duplication in imports, add index.ts file in each component folder that'll re-export it, e.g:
```export {Home} from './Home'```
Then you can import it as:
```import Home from "./components/Home/Home";```